### DISCUSSION TITLE
Aggregating data from many BirdNET-Pi installations

### DISCUSSION THREAD

**Discussion by Molokai** [mcguirepr89/BirdNET-Pi#719] [Ideas]
Has anyone looked at the possibility of adding support for having a centralized BirNET-Pi/website that doesn't (necessarily) do recordings, but instead provides aggregated information from multiple recording sites, as well as a mechanism to easily collect the data from multiple Pi's ?

I'd think that would be great for several reasons, not the least of which is to allow public assess to it without it impacting the actual recording machines - but also as part of better mapping of bird activity in a larger area.

**Comment by jds3d** [comment]
This seems similar the the functionality offered by birdweather.

**Comment by DemBones78** [comment]
i would rather see a VM / docker x86 version that can take many rstp audio streams from security cameras. many of the cameras out there have great mics Vs the limited hardware of a PI
